security/azure/sql_server_va_emails/sql_server_va_emails.pt

```diff
@@ -126,37 +329,77 @@ datasource "ds_sql_servers_with_va" do
     encoding "json"
     field "vaState", jmes_path(response, "properties.recurringScans.isEnabled")
     field "vaEmailSubscriptionAdmins", jmes_path(response, "properties.recurringScans.emailSubscriptionAdmins")
-    field "vaEmailList", jmes_path(response, "properties.recurringScans.emails")
+    field "vaEmails", jmes_path(response, "properties.recurringScans.emails")
     field "vaStorageContainerPath", jmes_path(response, "properties.storageContainerPath")
     field "id", val(iter_item, "id")
-    field "name", val(iter_item, "name")
-    field "location", val(iter_item, "location")
     field "kind", val(iter_item, "kind")
-    field "properties", val(iter_item, "properties")
+    field "region", val(iter_item, "region")
+    field "name", val(iter_item, "name")
     field "tags", val(iter_item, "tags")
-    field "subscriptionId", val(iter_item, "subscriptionId")
+    field "fqdn", val(iter_item, "fqdn")
+    field "state", val(iter_item, "state")
+    field "version", val(iter_item, "version")
+    field "resourceGroup", val(iter_item, "resourceGroup")
+    field "subscriptionID", val(iter_item, "subscriptionID")
     field "subscriptionName", val(iter_item, "subscriptionName")
   end
 end
 
-datasource "ds_bad_sql_servers" do
-  run_script $js_bad_sql_servers, $ds_sql_servers_with_va
+datasource "ds_bad_azure_sql_servers" do
+  run_script $js_bad_azure_sql_servers, $ds_sql_servers_with_va, $ds_applied_policy
 end
 
-###############################################################################
-# Scripts
-###############################################################################
-
-script "js_bad_sql_servers", type: "javascript" do
-  parameters "ds_sql_servers_with_va"
+script "js_bad_azure_sql_servers", type: "javascript" do
+  parameters "ds_sql_servers_with_va", "ds_applied_policy"
   result "result"
   code <<-EOS
-  result = []
+  bad_servers = _.filter(ds_sql_servers_with_va, function(db) {
+    return db['vaEmailList'] == '' || db['vaEmailList'] == null || db['vaEmailList'] == undefined || db['vaEmailList'].length == 0
```

Comment:
Should this be `vaEmails` as `vaEmailList` field no longer exists?

```suggestion
    return db['vaEmails'] == '' || db['vaEmails'] == null || db['vaEmails] == undefined || db['vaEmails'].length == 0
```

The same with lines 379 and 399